### PR TITLE
DOC: missing release notes updates

### PR DESCRIPTION
**Pull request by klauer**
Fixes previous PRs where I neglected to follow protocol (oops...)

I think this is it - hopefully.

### CODE REVIEW

**Comment by klauer** [comment]
@cristinasewell this is an example for what the others could include. Make sure you list yourself (github username) as a contributor in your document, though.

**Comment by cristinasewell** [comment]
> @cristinasewell this is an example for what the others could include. Make sure you list yourself (github username) as a contributor in your document, though.

Thanks @klauer - we're not supposed to manually edit it correct? or are we?
```
docs/pre-release-notes.sh 570 Old CCM Energy Calculation Reversion

source/upcoming_release_notes/570-Old.rst: line 1: IssueNumber: command not found
source/upcoming_release_notes/570-Old.rst: line 4: API: command not found
source/upcoming_release_notes/570-Old.rst: line 5: -----------: command not found
source/upcoming_release_notes/570-Old.rst: line 6: -: command not found
.........
```
I get something like that - am I missing some packages or is this normal? - i think i am doing something wrong here

**Comment by klauer** [comment]
Use underscores in your 2nd argument and it should fix it:
`docs/pre-release-notes.sh 570 'Old_CCM_Energy_Calculation_Reversion'`

**Comment by cristinasewell** [comment]
> docs/pre-release-notes.sh 570 'Old_CCM_Energy_Calculation_Reversion

thank you

File: docs/source/upcoming_release_notes/311-valueerror_typo.rst [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+- Fixed a typo in a ``ValueError`` exception in
+  :meth:`pcdsdevices.state.StatePositioner.check_value`.
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- N/A
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/311-valueerror_typo.rst:31]:
```suggestion
- klauer
```

File: docs/source/upcoming_release_notes/380-rbv_read_only.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  :class:`pcdsdevices.gauge.GaugeSerialGPI`, component ``autozero``.
+  All other devices were audited, finding no other RBV-related read-only items.
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- N/A
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/380-rbv_read_only.rst:32]:
```suggestion
- klauer
```

File: docs/source/upcoming_release_notes/511-at2l0_interface.rst [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+--------
+- N/A
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- N/A
```
Comment (ZLLentz) [inline docs/source/upcoming_release_notes/511-at2l0_interface.rst:31]:
```suggestion
- klauer
```

**Review by ZLLentz — APPROVED** [review]

**Pull request merged**